src/index.ts

```diff
@@ -0,0 +1,25 @@
+import A11yDialog from './a11y-dialog'
+
+function instantiateDialogs() {
+  for (const el of document.querySelectorAll('[data-a11y-dialog]')) {
+    new A11yDialog(el as HTMLElement)
+  }
+}
+
+if (typeof document !== 'undefined') {
+  if (document.readyState === 'loading') {
+    document.addEventListener('DOMContentLoaded', instantiateDialogs)
+  } else {
+    instantiateDialogs()
+  }
+}
+
+// TypeScript doesn’t know about the `inert` attribute/property yet; this
+// declaration extends the `HTMLElement` interface to include it
+declare global {
+  interface HTMLElement {
+    inert: boolean
+  }
+}
```

Comment:
```suggestion
```

We can delete this because we're not referencing inert-as-a-DOM-property anymore :)